docs/04-operation-guides/troubleshooting/evnt-02-jetstream-troubleshooting.md

```diff
@@ -0,0 +1,50 @@
+---
+title: JetStream backend troubleshooting
+---
+
+## Symptom
+
+Events were not received by the consumers.
+
+## Remedy
+
+1. Follow the basic troubleshooting steps as mentioned in [Eventing Troubleshooting](./evnt-01-eventing-troubleshooting.md) guide.
+
+2. Use the [nats CLI](https://github.com/nats-io/natscli) to check if the stream was created:
+
+    1. Port forward the Kyma Eventing NATS Service to localhost. Use port `4222`. Run:
+        ```bash
+        kubectl -n kyma-system port-forward svc/eventing-nats 4222
+        ```
+    2. Use the [nats CLI](https://github.com/nats-io/natscli) to list the streams:
+       ```bash
+       nats stream ls
+       ```
+    
+        ```bash
+        ╭────────────────────────────────────────────────────────────────────────────╮
+        │                                  Streams                                   │
+        ├──────┬─────────────┬─────────────────────┬──────────┬───────┬──────────────┤
+        │ Name │ Description │ Created             │ Messages │ Size  │ Last Message │
+        ├──────┼─────────────┼─────────────────────┼──────────┼───────┼──────────────┤
+        │ sap  │             │ 2022-05-03 00:00:00 │ 0        │ 318 B │ 5.80s        │
+        ╰──────┴─────────────┴─────────────────────┴──────────┴───────┴──────────────╯
+        ```       
```

Comment:
Is there a specific reason to split it across 2 bash blocks ? See [here](https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown-files) for shell vs bash.

```shell
$ nats stream ls
╭────────────────────────────────────────────────────────────────────────────╮
│                                  Streams                                   │
├──────┬─────────────┬─────────────────────┬──────────┬───────┬──────────────┤
│ Name │ Description │ Created             │ Messages │ Size  │ Last Message │
├──────┼─────────────┼─────────────────────┼──────────┼───────┼──────────────┤
│ sap  │             │ 2022-05-03 00:00:00 │ 0        │ 318 B │ 5.80s        │
╰──────┴─────────────┴─────────────────────┴──────────┴───────┴──────────────╯
```